src/app/shared/spotlight-card/spotlight-card.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-spotlight-card',
+  templateUrl: './spotlight-card.component.html',
+  styleUrls: ['./spotlight-card.component.scss'],
+})
+export class SpotlightCardComponent  {
+  @Input() spotlightstory:any;
+  @Input() index:any;
+  @Output() emitstory = new EventEmitter<any>();
+  constructor() {  }
+
+  emitStory(storyid:any){
```

Comment:
Do we need to emit the event?
Can't we write the redirection logic within this component itself?

Reply:
we can redirect and the logic changed